From: Ross Vandegrift <ross@kallisti.us>
Date: Sat, 22 Mar 2014 21:56:45 +0000
Subject: Disable tests incompatible with Tesseract 3.3.0.

Signed-off-by: Ross Vandegrift <ross@kallisti.us>
---
 tests/tests_tesseract.py | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/tests/tests_tesseract.py b/tests/tests_tesseract.py
index 869522f..0d6e016 100644
--- a/tests/tests_tesseract.py
+++ b/tests/tests_tesseract.py
@@ -22,6 +22,8 @@ class TestContext(unittest.TestCase):
         self.assertTrue(tesseract.is_available(),
                        "Tesseract not found. Is it installed ?")
 
+    @unittest.skipIf(tesseract.get_version() != (3, 2, 1),
+                     "This test only works with Tesseract 3.02.1")
     def test_version(self):
         self.assertEqual(tesseract.get_version(), (3, 2, 1),
                          ("Tesseract does not have the expected version"
@@ -71,9 +73,13 @@ class TestTxt(unittest.TestCase):
     def test_basic(self):
         self.__test_txt('test.png', 'test.txt')
 
+    @unittest.skipIf(tesseract.get_version() != (3, 2, 1),
+                     "This test only works with Tesseract 3.02.1")
     def test_european(self):
         self.__test_txt('test-european.jpg', 'test-european.txt')
 
+    @unittest.skipIf(tesseract.get_version() != (3, 2, 1),
+                     "This test only works with Tesseract 3.02.1")
     def test_french(self):
         self.__test_txt('test-french.jpg', 'test-french.txt', 'fra')
 
@@ -118,6 +124,8 @@ class TestCharBox(unittest.TestCase):
     def test_french(self):
         self.__test_txt('test-french.jpg', 'test-french.box', 'fra')
 
+    @unittest.skipIf(tesseract.get_version() != (3, 2, 1),
+                     "This test requires Tesseract 3.02.1")
     def test_japanese(self):
         self.__test_txt('test-japanese.jpg', 'test-japanese.box', 'jpn')
 
@@ -190,6 +198,8 @@ class TestWordBox(unittest.TestCase):
     def test_french(self):
         self.__test_txt('test-french.jpg', 'test-french.words', 'fra')
 
+    @unittest.skipIf(tesseract.get_version() != (3, 2, 1),
+                     "This test requires Tesseract 3.02.1")
     def test_japanese(self):
         self.__test_txt('test-japanese.jpg', 'test-japanese.words', 'jpn')
 
@@ -256,6 +266,8 @@ class TestLineBox(unittest.TestCase):
     def test_french(self):
         self.__test_txt('test-french.jpg', 'test-french.lines', 'fra')
 
+    @unittest.skipIf(tesseract.get_version() != (3, 2, 1),
+                     "This test requires Tesseract 3.02.1")
     def test_japanese(self):
         self.__test_txt('test-japanese.jpg', 'test-japanese.lines', 'jpn')
 
