From: Ross Vandegrift <ross@kallisti.us>
Date: Sat, 22 Mar 2014 21:48:16 +0000
Subject: Fix missing comma from export list.

Signed-off-by: Ross Vandegrift <ross@kallisti.us>
---
 src/pyocr/tesseract.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pyocr/tesseract.py b/src/pyocr/tesseract.py
index a8576df..f1fef4c 100755
--- a/src/pyocr/tesseract.py
+++ b/src/pyocr/tesseract.py
@@ -46,7 +46,7 @@ __all__ = [
     'CharBoxBuilder',
     'get_available_builders',
     'get_available_languages',
-    'get_name'
+    'get_name',
     'get_version',
     'image_to_string',
     'is_available',
